### PR TITLE
OAS-4513 | Warn about root credential usage / root password rotation feature

### PR DESCRIPTION
**Pull request by Simran-B**
Waiting for text adjustments in Oasis dashboard to take a new screenshot.

https://deploy-preview-937--zealous-morse-14392b.netlify.app/docs/3.8/oasis/deployments.html#how-to-enable-the-root-user-password-rotation

### CODE REVIEW

File: oasis/deployments.md [hunk 1]
```diff
@@ -166,6 +172,19 @@ and automatically accepted by most web browsers. The certificate used on port
 a self-signed certificate is recommended.
 {% endhint %}
 
+## How to enable the root user password rotation
+
+1. Navigate to the __Deployment__ for which you want to enable an automatic
+   password rotation for the root user.
+2. On the __Overview__ tab, click the button with the __cog__ icon next to the
```
Comment (ansoboleva) [inline oasis/deployments.md:179]: Please use the "gear icon" instead of "cog".

File: oasis/deployments.md [hunk 2]
```diff
@@ -166,6 +172,19 @@ and automatically accepted by most web browsers. The certificate used on port
...
+   __ROOT PASSWORD__.
+3. __Password Settings__ dialog, turn the password rotation on and click the
```
Comment (ansoboleva) [inline oasis/deployments.md:181]: In the Password Settings dialog...

File: oasis/deployments.md [hunk 3]
```diff
@@ -166,6 +172,19 @@ and automatically accepted by most web browsers. The certificate used on port
 a self-signed certificate is recommended.
 {% endhint %}
 
+## How to enable the root user password rotation
```
Comment (ansoboleva) [inline oasis/deployments.md:175]: Please add a couple word describing what this feature is about.

File: oasis/deployments.md [hunk 4]
```diff
@@ -166,6 +172,19 @@ and automatically accepted by most web browsers. The certificate used on port
...
+
+1. Navigate to the __Deployment__ for which you want to enable an automatic
+   password rotation for the root user.
+2. On the __Overview__ tab, click the button with the __cog__ icon next to the
+   __ROOT PASSWORD__.
+3. __Password Settings__ dialog, turn the password rotation on and click the
+   __Confirm__ button.
+
+   ![Oasis Deployment Password Rotation](images/oasis-deployment-password-rotation.png)
```
Comment (ansoboleva) [inline oasis/deployments.md:184]: We need to adjust the message displayed on the GUI:

Opt-in for password rotation for the root user --> Enable password rotation for the root user

Description --> When enabled, the root password will be automatically rotated every three months. You'll be notified about the rotation by email. When this option is turned off, the root password will not be rotated.

**Review by ansoboleva — CHANGES_REQUESTED** [review]
Please see the comments.

File: oasis/deployments.md (hunk 4 above)
Comment (Simran-B) [reply oasis/deployments.md:184]: @nikhil-varma: Can you take care of it? The source code appears to be here:

- https://github.com/arangodb-managed/dashboard/blob/master/frontend/src/dashboard/deployment/OverviewViews.tsx#L245
- https://github.com/arangodb-managed/dashboard/blob/4ead56e9ad25c819e51f23082eb7a31f3a3a1d97/frontend/src/dashboard/deployment/PasswordSettingsModalView.tsx#L93-L101

File: oasis/deployments.md (hunk 2 above)
Comment (Simran-B) [inline oasis/deployments.md:181]:
```suggestion
3. In the __Password Settings__ dialog, turn the password rotation on and click the
```

File: oasis/deployments.md (hunk 1 above)
Comment (Simran-B) [inline oasis/deployments.md:179]:
```suggestion
2. On the __Overview__ tab, click the button with the __gear__ icon next to the
```

File: oasis/deployments.md (hunk 4 above)
Comment (ansoboleva) [reply oasis/deployments.md:189]: @Simran-B we adjusted the texts with Nikhil, the GUI will be updated soon.

**Review by ansoboleva — APPROVED** [review]
please don't forget to update the image, otherwise lgtm

**Pull request merged**